configs/x86_64-zephyr-linux-gnu.config

```diff
@@ -0,0 +1,21 @@
+CT_CONFIG_VERSION="2"
+CT_LOCAL_TARBALLS_DIR="${CT_PREFIX:-${HOME}/x-tools}/sources"
+# CT_PREFIX_DIR_RO is not set
+CT_PATCH_BUNDLED_LOCAL=y
+CT_LOCAL_PATCH_DIR="${CT_TOP_DIR}/../../patches"
+# CT_LOG_PROGRESS_BAR is not set
+CT_ARCH_X86=y
+CT_MULTILIB=y
+CT_ARCH_64=y
+CT_TARGET_VENDOR="zephyr"
+CT_KERNEL_LINUX=y
+CT_BINUTILS_LINKER_LD_GOLD=y
```

Comment:
any particular reason to use GOLD?

Reply:
Most of the config options here are copied straight from `crosstool-ng/samples/x86_64-multilib-linux-gnu/crosstool.config`. If you want, I can try without gold.

Reply:
Yes, lets try w/o gold.

Reply:
Updated patch to remove gold.